core/flyout.js

```diff
@@ -427,9 +581,14 @@ Blockly.Flyout.prototype.show = function(xmlList) {
   this.listeners_.push(Blockly.bindEvent_(this.svgBackground_, 'mouseover',
       this, deselectAll));
 
-  this.width_ = 0;
+  if (this.horizontalLayout_) {
```

Comment:
This isn't during a scroll.  It's in show().  It gets reset here and then set in reflow().